src/native_main.nim

```diff
@@ -233,6 +296,8 @@ proc handleMessage(msg: MessageRecv): string =
 
     return $ %* reply # $ converts to string, %* converts to JSON
 
+parseCommandLineOptions()
```

Comment:
If commandlineparams doesn't exist on all systems, why are we trying to execute this on all systems? It will spam the Firefox error console with warnings that will never be meaningful since Tridactyl will never launch the messenger with command line parameters